setup.sh

```diff
@@ -397,6 +397,25 @@ if continueYesNo "$ask"; then
   fi
 fi
 
+# We install the IDEs last just o be sure we did not miss any dependency
+ask="Install: slack-desktop?"
+if continueYesNo "$ask"; then
+    runCommand "sudo snap install slack --classic"
+    runCommand "sudo apt-get update"
+    runCommand "sudo apt-get upgrade slack-desktop"
+    output "SUCCESS!"
+fi
+
+# We install the IDEs last just o be sure we did not miss any dependency
```

Comment:
And this comment, but to be honest vscode is a better sublime replacement and even atom but it can be here :)